### PR TITLE
Audiobook manifest improvements

### PR DESCRIPTION
**Pull request by leonardr**
Previously, Findaway-derived audiobook manifests looked like this:

```
{
   "spine" : [
      {
         "type" : null,
         "duration" : 16.201,
         "findaway:part" : 0,
         "href" : null,
         "title" : "Track 1",
         "findaway:sequence" : 1
      },
       ...
      {
         "href" : null,
         "findaway:sequence" : 79,
         "title" : "Track 79",
         "findaway:part" : 0,
         "duration" : 32.318,
         "type" : null
      }
   ],
   "metadata" : {
      "@type" : "http://bib.schema.org/Audiobook",
      "language" : "en",
      "title" : "2001",
      "identifier" : "http://www.gutenberg.org/ebooks/2002",
      "findaway:fulfillmentId" : "123456",
      "duration" : 28371.256,
      "authors" : [
         "2003"
      ],
      "findaway:sessionKey" : "aaaaaaaa-4444-cccc-dddd-666666666666",
      "findaway:accountId" : "3M",
      "findaway:checkoutId" : "abcdef01234789abcdef0123",
      "findaway:licenseId" : "1234567890987654321ababa"
   },
   "@context" : [
      "http://readium.org/webpub/default.jsonld",
      {
         "findaway" : "http://librarysimplified.org/terms/third-parties/findaway.com/"
      }
   ]
}
```

Now, they look like this:

```
{
   "spine" : [
      {
         "title" : "Track 1",
         "findaway:part" : 0,
         "href" : null,
         "type" : "audio/mpeg",
         "findaway:sequence" : 1,
         "duration" : 16.201
      },
       ...
      {
         "duration" : 32.318,
         "findaway:sequence" : 79,
         "href" : null,
         "title" : "Track 79",
         "findaway:part" : 0,
         "type" : "audio/mpeg"
      }
   ],
   "@context" : [
      "http://readium.org/webpub/default.jsonld",
      {
         "findaway" : "http://librarysimplified.org/terms/third-parties/findaway.com/"
      }
   ],
   "metadata" : {
      "identifier" : "http://www.gutenberg.org/ebooks/2002",
      "title" : "2001",
      "language" : "en",
      "duration" : 28371.256,
      "authors" : [
         "2003"
      ],
      "@type" : "http://bib.schema.org/Audiobook",
      "encrypted" : {
         "findaway:licenseId" : "1234567890987654321ababa",
         "findaway:checkoutId" : "abcdef01234789abcdef0123",
         "findaway:accountId" : "3M",
         "findaway:fulfillmentId" : "123456",
         "findaway:sessionKey" : "aaaaaaaa-4444-cccc-dddd-666666666666"
      }
   }
}
```

* The DRM fields are collected in a separate 'encrypted' object rather than being individual 'metadata' items.
* For the sake of maximum compatibility with `audiobook/manifest+json`, the media type of each item is included (but it doesn't matter because the client doesn't use this information).
* The spine items are sorted by `findaway:part` and then by `findaway:sequence`.

### CODE REVIEW

File: api/feedbooks.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def __init__(self, _db, collection, *args, **kwargs):
 
         self.new_css = None
         if new_css_url and self.http_get:
-            status_code, headers, content = self.http_get(new_css_url)
+            status_code, headers, content = self.http_get(new_css_url, {})
```
Comment (leonardr) [inline api/feedbooks.py]: An unrelated change that caused a real-world Feedbooks import to fail even though it worked when `http_get` was mocked.

File: api/bibliotheca.py [hunk 2]
```diff
@@ -368,14 +371,10 @@ def findaway_license_to_webpub_manifest(
         else:
             logging.error("Unknown Findaway audio format encountered: %s",
                           audio_format)
+            part_media_type = None
 
-        # TODO: The items are in an ordered list, but each one also
```
Comment (courte) [inline api/bibliotheca.py]: More like "TODONE", amirite?
